File: src/main/scala/com/mogobiz/sample/config/SampleCake.scala
package com.mogobiz.sample.config

import com.mogobiz.pay.handlers.shipping.EasyPostHandler
import com.mogobiz.run.config.DefaultMogobizCake
import com.mogobiz.sample.handlers.MyCouponHandler

class SampleCake extends DefaultMogobizCake {
  println("HOUHOU")
  println("HOUHOU")
  println("HOUHOU")
  println("HOUHOU")
  println("HOUHOU")
  println("HOUHOU")
  println("HOUHOU")
  println("HOUHOU")
  println("HOUHOU")
  override val easyPostHandler = new EasyPostHandler

  override val couponHandler = new MyCouponHandler

}

File: src/main/java/com/mogobiz/samplej7/config/SampleCake.java
package com.mogobiz.samplej7.config;

import com.mogobiz.pay.handlers.shipping.EasyPostHandler;
import com.mogobiz.run.config.DefaultMogobizCake;
import com.mogobiz.run.handlers.CouponHandler;
import com.mogobiz.samplej7.handlers.MyCouponHandler;

public class SampleCake extends DefaultMogobizCake {
  @Override
  public EasyPostHandler easyPostHandler() {
    return null;
  }

  @Override
  public CouponHandler couponHandler() {
    return new MyCouponHandler();
  }

}
